exercicios/para-casa/biblioteca_pg2.py

```diff
@@ -0,0 +1,37 @@
+from livro import Livro
+
+class Biblioteca:
+    def __init__(self):
+        self.livros = []
+        
+    def adicionar_livro(self, livro: Livro):
+        if(not isinstance(livro, Livro)):
+            raise TypeError(f"Esperado livro obtido valor {livro} do tipo {type(livro)}")
+        self.livros.append(livro)
+
+    def exibir_livros(self, livro: Livro):
+        #if 
+        return self.livros
+    
+    def emprestar_livro(self, livro):
+        if livro in self.livros:
+            return True
+        else:
+            return False
+    
+"""
+    def buscar_livro(self, livro: Livro):
+        if(not isinstance(livro, Livro)):
+            raise TypeError("Esse livro não faz parte do nosso acervo")
+        return self.livros
+    
+
+
+    def buscar_livros(self):
+        id_livro = input("Digite o nome do livro desejado:")
```

Comment:
Vi que essa implementação do método buscar_livros está bem promissora! 

Só um comentário que não é necessário ter esse input do teclado aqui dentro da classe, acho quer não chegaram nessa parte da explicação de Orientação a objetos mas existe um conceito de responsabilidades da classe, esse método é responsável apenas por buscar o item na lista, aqui ele está lendo um input do teclado também. Isso pode ser feito por quem for chamar a classe biblioteca e o método de buscar_livros

Mas achei muito bacana que está trabalhando nessa parte